devito/dse/rewriters.py

```diff
@@ -411,10 +411,16 @@ class CustomRewriter(AggressiveRewriter):
 
     passes_mapper = {
         'extract_sum_of_products': AggressiveRewriter._extract_sum_of_products,
-        'factorization': AggressiveRewriter._factorize,
+        'factorize': AggressiveRewriter._factorize,
         'eliminate_inter_stencil_redundancies':
-            AggressiveRewriter._eliminate_inter_stencil_redundancies
-
+            AggressiveRewriter._eliminate_inter_stencil_redundancies,
+        'eliminate_intra_stencil_redundancies':
```

Comment:
why the difference between `inter` and `intra`? couldn't it be the same in both?